packages/runtime/container-runtime/src/test/summarizer.spec.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 import { strict as assert } from "assert";
-import { SummarizerStopReason } from "@fluidframework/container-runtime-previous";
```

Comment:
Do we know what this was about?

Reply:
Yes, I sync up with Kian and he let me know it was not on purpose (VS Code auto suggestion).